src/stringProcessing/getSubheadings.js

```diff
@@ -1,16 +1,35 @@
-import { map } from "lodash-es";
-
 /**
  * Gets all subheadings from the text and returns these in an array.
  *
  * @param {string} text The text to return the headings from.
- * @returns {Array} Matches of subheadings in the text, first key is everything including tags, second is the heading
- *                  level, third is the content of the subheading.
+ *
+ * @returns {Array<string[]>} Matches of subheadings in the text, first key is everything including tags,
+ *                            second is the heading level, third is the content of the subheading.
  */
 function getSubheadings( text ) {
```

Comment:
Perhaps we can rename this into `getHeadings`?
Since this also gathers all `h1` headings as well.

Reply:
Interesting, but after recalibration takes over this function will no longer be used. We might remove it then. So I do not think it's worth renaming now.